src/GameLogic/PlayerActions/Character/IncreaseStatsAction.cs

```diff
@@ -24,16 +24,19 @@ public void IncreaseStats(Player player, AttributeDefinition statAttributeDefini
             throw new InvalidOperationException("No character selected");
         }
 
-        if (player.SelectedCharacter.LevelUpPoints > 0)
+        var selectedCharacter = player.SelectedCharacter;
+        if (selectedCharacter.CharacterStatus != CharacterStatus.GameMaster && player.SelectedCharacter?.LevelUpPoints <= 0)
         {
-            var attributeDef = player.SelectedCharacter.CharacterClass?.StatAttributes.FirstOrDefault(a => a.Attribute == statAttributeDefinition);
-            if (attributeDef is { IncreasableByPlayer: true })
-            {
-                player.Attributes![attributeDef.Attribute]++;
-                player.SelectedCharacter.LevelUpPoints--;
-                player.ViewPlugIns.GetPlugIn<IStatIncreaseResultPlugIn>()?.StatIncreaseResult(statAttributeDefinition, true);
-                return;
-            }
+            return;
```

Comment:
Because of this return, no result is sent anymore.

Reply:
Fixed